canonical/ubuntu.com
# add expandable navigation to core docs

## Done

- This feature makes side navigation expandable. 

> Note: The data i used in first screenshot is from https://microk8s.io/docs, the second one is from the current API that should add home page as first level like we have in microk8s.io

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/core/docs
    - Be sure to test on mobile, tablet and desktop screen sizes

## Issue / Card

Fixes #12026 

## Screenshots
**Image 1**
![image-1](https://user-images.githubusercontent.com/20561190/191013383-8ac75796-1480-427d-a1bc-6b819673e785.png)

**Image 2**
![image-2](https://user-images.githubusercontent.com/20561190/191013410-eb6fb8c8-1eea-4b05-bf5a-76c31a0685d3.png)


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
